phpunit.xml.dist

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<phpunit bootstrap="./tests/bootstrap.php"
```

Comment:
How would it load without including bootstrap that has composer autoload required?